src/view_element.cpp

```diff
@@ -47,6 +47,51 @@
 #include "verse.h"
 #include "vrv.h"
 
+/**
+  * Translates a MEI head.shape attribute value to a SMuFL glph code
+  * (see http://music-encoding.org/documentation/3.0.0/data.HEADSHAPE.list)
+  * Note that only a few of the strings have been mapped thus far.
+ **/
+/*wchar_t getFontNumber(std::string s)
+{
+    if (s == "filldiamond") return vrv::SMUFL_E0DB_noteheadDiamondBlack;
+    if (s == "diamond") return vrv::SMUFL_E0DD_noteheadDiamondWhite;
+    if (s == "fillisotriangle") return vrv::SMUFL_E0BE_noteheadTriangleUpBlack;
+    if (s == "isotriangle") return vrv::SMUFL_E0BD_noteheadTriangleUpWhite;
+    if (s == "x") return vrv::SMUFL_E0A9_noteheadXBlack;
+    if (s == "circlex") return vrv::SMUFL_E0B3_noteheadCircleX;
+    return 0;
+}
+*/
+wchar_t getFontNumber(vrv::data_HEADSHAPE_list s)
```

Comment:
As commented in the previous PR we can move this function to a Note::GetHeadShapeGlyph method with no parameter. vrv:: prefixes should be removed